libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -934,16 +934,21 @@ void ofFbo::updateTexture(int attachmentPoint) {
 			glPushAttrib(GL_COLOR_BUFFER_BIT);
 		}
 
-		bind();
+		// Because we temporarily change global Fbo binding state
+		// when blitting from one framebuffer to the other, we first have 
+		// to query the renderer for its binding state, and store it.
+		GLuint rendererFboBindingState = ofGetGLRenderer()->getCurrentFramebufferId();
```

Comment:
yeah it makes sense that the renderer keeps track of the bound buffers... but the way it's implemented is kind of weird, i'm trying to change it to remove the state from the fbo and completely move it to the renderer. one question, in this change you are never setting back the read framebuffer and draw framebuffer to the previous value, is that correct?